examples/dicklang/parse_test.go

```diff
@@ -17,13 +17,13 @@ func TestParser(t *testing.T) {
 	mod, err := Parse("sample.dicklang", []rune(src))
 	require.NoError(t, err)
 	require.Len(t, mod.Dicks, 9)
-	require.Equal(t, mod.Dicks[0].Frag.Src(), []rune("B===>"))
-	require.Equal(t, mod.Dicks[1].Frag.Src(), []rune("8==>"))
-	require.Equal(t, mod.Dicks[2].Frag.Src(), []rune("B::>"))
-	require.Equal(t, mod.Dicks[3].Frag.Src(), []rune("<====8"))
-	require.Equal(t, mod.Dicks[4].Frag.Src(), []rune("<::::::3"))
-	require.Equal(t, mod.Dicks[5].Frag.Src(), []rune("8xxxx>"))
-	require.Equal(t, mod.Dicks[6].Frag.Src(), []rune("8xxx=xxx>"))
-	require.Equal(t, mod.Dicks[7].Frag.Src(), []rune("B:x:=:x>"))
-	require.Equal(t, mod.Dicks[8].Frag.Src(), []rune("<:=3"))
+	require.Equal(t, []rune("B===>"), mod.Dicks[0].Frag.Src())
```

Comment:
It probably makes more sense to compare against strings and turn the actual slices into strings like this:

```
require.Equal(t, "B===>", string(mod.Dicks[0].Frag.Src()))
```

because the diffs are human-readable in this case: https://play.golang.org/p/JxHUNtvFbK5